app/controllers/submissions_controller.rb

```diff
@@ -114,6 +114,14 @@ def repo_browser
       end
     end
 
+    last_rev = @grouping.submissions
+    @last_submission = nil
+    if !last_rev.empty?
+      selected = @revisions_history.select { |rev| rev[:num] == 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Trailing whitespace detected.